src/main/java/janggi/coordinate/Route.java

```diff
@@ -0,0 +1,59 @@
+package janggi.coordinate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Route {
+
+    private final Position departure;
+    private final Position destination;
+
+    private Route(final Position departure, final Position destination) {
+        this.departure = departure;
+        this.destination = destination;
+    }
+
+    public static Route of(final Position departure, final Position destination) {
+        return new Route(departure, destination);
+    }
+
+    public List<Position> calculate() {
+        return calculate(true);
+    }
+
+    public List<Position> calculate(boolean excludeDestination) {
```

Comment:
```suggestion
    public List<Position> calculateIncludeDestination() {
        return calculate(true);
    }

    public List<Position> calculateExcludeDestination() {
        return calculate(false);
    }
    
    private List<Position> calculate(boolean excludeDestination) {
```
이렇게도 작성해볼 수 있을 것 같네요

Reply:
감사합니다! 궁성 구현에서 출발지도 포함되어야하는 경우가 생겨서 이런 방법으로 리팩토링해뒀었는데, 훨씬 명확하다고 느꼈습니다.